doc/input-output-reference/src/overview/group-simulation-parameters.tex

```diff
@@ -562,7 +562,7 @@ \subsubsection{Inputs}\label{inputs-10-019}
 
 \paragraph{Field: Polygon Clipping Algorithm}\label{field-polygon-clipping-algorithm}
 
-This is an advanced feature. Prior to V7, the internal polygon clipping method was a special case of the Weiler-Atherton method. Now, two options are available: \textbf{SutherlandHodgman} (default) and \textbf{ConvexWeilerAtherton}. Theoretically, Sutherland-Hodgman is a simpler algorithm but it works well in cases where receiving surfaces (of shadows) are non-convex. The Weiler-Atherton implementation is only accurate where both casting and receiving surfaces are convex. Warnings/severe errors are displayed when necessary. More details on polygon clipping are contained in the Engineering Reference.
+This is an advanced feature. Prior to V7, the internal polygon clipping method was a special case of the Weiler-Atherton method. Now, three options are available: \textbf{SutherlandHodgman} (default), \textbf{SlaterBarskyandSutherlHodgman} and \textbf{ConvexWeilerAtherton}. Theoretically, Sutherland-Hodgman is a simpler algorithm but it works well in cases where receiving surfaces (of shadows) are non-convex. The Weiler-Atherton implementation is only accurate where both casting and receiving surfaces are convex. Warnings/severe errors are displayed when necessary. If `SlaterBarskyandSutherlHodgman` is chosen, the polygon clipping for rectangular surfaces will be calculated using the *Slater and Barsky* algorithm, while the rest adopts the default *Sutherl and Hodgman* algorithm. More details on polygon clipping are contained in the Engineering Reference.
```

Comment:
The formatting of the option names here is inconsistent. The pdf renders some in bold, some in single quotes, some in *'s (mixing md and latex?). This would be much better using the `{description}` method, example [here](https://github.com/NREL/EnergyPlus/blob/55f5f74c4f7b5b99e0110e5c7a158fd85bb20cc2/doc/input-output-reference/src/overview/group-surface-construction-elements.tex#L4083-L4088).